symfony/plugins/orangehrmAdminPlugin/entity/License.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace OrangeHRM\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * License
+ *
+ * @ORM\Table(name="ohrm_license")
+ * @ORM\Entity
+ */
+class License
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="name", type="string", length=100)
+     */
+    private $name;
+
+    //mappedBy = license , ColumnName = "license_id"
+
+    /**
+     * @var Collection
+     *
+     * @ORM\ManyToMany(targetEntity="EmployeeLicense", mappedBy="License")
+     * @ORM\JoinColumns({
+     *   @ORM\JoinColumn(name="id", referencedColumnName="license_id")
+     * })
+     */
+    private $EmployeeLicense;
```

Comment:
remove this bidirectional mapping & the constructor. once remove test properly

Reply:
done, tested using postman and passed all test cases